activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/ResourceFinderDescriptor.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.spring.process;
+
+import java.util.List;
+
+public interface ResourceFinderDescriptor {
+
+    public List<String> getLocationSuffixes();
+    public String getLocationPrefix();
+    public boolean isCheckResources();
```

Comment:
Let's use a more meaningful name for this method: what about `shouldDiscoverResources` or `shouldLookUpResources`?